### PR TITLE
Gate Lifter

### PR DESCRIPTION
**Pull request by freezy**
This PR adds a gate lifter component, that enables and disables a gate through a coil.

[Documentation](https://github.com/freezy/VisualPinball.Engine/blob/feature/gate-lifter/VisualPinball.Unity/Documentation%7E/creators-guide/manual/mechanisms/lifting-gates.md)

This PR also:
- Fixes a bug in the score reel inspector
- Adds a *direction* toggle for score reels.
- Fixes a bug in the wire- and coil player (thanks @jsm174).

### CODE REVIEW

File: VisualPinball.Unity/VisualPinball.Unity/Display/ScoreReelComponent.cs [hunk 1]
```diff
@@ -23,6 +23,14 @@ namespace VisualPinball.Unity
 {
 	public class ScoreReelComponent : MonoBehaviour
 	{
+		public enum ScoreReelDirection
+		{
+			up, down
```
Comment (jsm174) [inline VisualPinball.Unity/VisualPinball.Unity/Display/ScoreReelComponent.cs:28]: I looked pretty quick (so possibly I missed), but all the enums in the project I saw always begin with a capital letter.

File: VisualPinball.Unity/VisualPinball.Unity.Editor/VPT/Gate/GateLifterInspector.cs [hunk 2]
```diff
@@ -0,0 +1,44 @@
...
+{
+	[CustomEditor(typeof(GateLifterComponent)), CanEditMultipleObjects]
+	public class GateLifterInspector : UnityEditor.Editor
+	{
+		private SerializedProperty _angleProperty;
+		private SerializedProperty _speedProperty;
+
+		private void OnEnable()
+		{
+			_angleProperty = serializedObject.FindProperty(nameof(GateLifterComponent.LiftedAngleDeg));
```
Comment (jsm174) [inline VisualPinball.Unity/VisualPinball.Unity.Editor/VPT/Gate/GateLifterInspector.cs:30]: Not really an issue, but "Lifted" in the variable name vs "Lifting" in the text.

**Review by jsm174 — APPROVED** [review]

File: VisualPinball.Unity/VisualPinball.Unity/Display/ScoreReelComponent.cs (hunk 1 above)
Comment (freezy) [reply VisualPinball.Unity/VisualPinball.Unity/Display/ScoreReelComponent.cs:28]: Changed!

File: VisualPinball.Unity/VisualPinball.Unity.Editor/VPT/Gate/GateLifterInspector.cs (hunk 2 above)
Comment (freezy) [reply VisualPinball.Unity/VisualPinball.Unity.Editor/VPT/Gate/GateLifterInspector.cs:30]: Changed!

**Comment by freezy** [comment]
Thanks for the review!

**Pull request merged**